### PR TITLE
Update Gitpod Config to properly configure Julia

### PR DESCRIPTION
**Pull request by JesterOrNot**
Hello! :smile:

I saw you had a Gitpod config and also that Julia wasn't properly installed so I created this PR with Julia properly configured!

## Before

image

## After

image

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+# Install Julia
+RUN sudo apt-get update \
+    && sudo apt-get install -y \
+        build-essential \
```
Comment (jankeromnes) [inline .gitpod.Dockerfile:8]: Note: `build-essential` is already installed in `gitpod/workspace-full`:

https://github.com/gitpod-io/workspace-images/blob/b17e2ea52123f14c26ddb99e0a9f2f233312bf25/full/Dockerfile#L9

File: .gitpod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        libatomic1 \
+        python \
```
Comment (jankeromnes) [inline .gitpod.Dockerfile:10]: Note:  `python` is already installed in `gitpod/workspace-full`:

https://github.com/gitpod-io/workspace-images/blob/b17e2ea52123f14c26ddb99e0a9f2f233312bf25/full/Dockerfile#L208-L221

File: .gitpod.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        gfortran \
+        perl \
+        wget \
+        m4 \
+        cmake \
```
Comment (jankeromnes) [inline .gitpod.Dockerfile:15]: Note:  `cmake` is already installed in `gitpod/workspace-full`:

https://github.com/gitpod-io/workspace-images/blob/b17e2ea52123f14c26ddb99e0a9f2f233312bf25/full/Dockerfile#L117-L118

File: .gitpod.Dockerfile (hunk 1 above)
Comment (JesterOrNot) [reply .gitpod.Dockerfile]: Done!

File: .gitpod.Dockerfile (hunk 2 above)
Comment (JesterOrNot) [reply .gitpod.Dockerfile]: Done!

File: .gitpod.Dockerfile (hunk 3 above)
Comment (JesterOrNot) [reply .gitpod.Dockerfile]: Done!

**Comment by dhairyagandhi96** [comment]
Do we know how often they update the Julia version?

**Comment by JesterOrNot** [comment]
Do you mean apt? I'm not sure, but I'd imagine frequently.

**Comment by CarloLucibello** [comment]
thanks!

**Pull request merged**